src/riak_ql_ddl.erl

```diff
@@ -179,22 +180,44 @@ get_local_key(?DDL{table = T}=DDL, Obj)
     Mod = make_module_name(T),
     get_local_key(DDL, Obj, Mod).
 
+-spec desc_adjusted_key([any()], module(), ?DDL{}) ->
```

Comment:
`lk_to_pk` already returns unnegated key (that is, `[1,2,3]`, not `[1,2,-3]`).

There are consumers of both DESC-adjusted LK and PK. If I were to consider tucking `desc_adjusted_key` into `lk_to_pk`, it would be linguistically problematic to arrive at a function name to express both unnegation of LK and LK-PK conversion. Exporting two, simpler, functions is a way to keep sane names for the entities and operations involved.

I also did the rename as suggested and swapped 2nd and 3rd parameters of `lk_to_pk` for consistency with others.